packages/matplotlib/src/wasm_backend.py

```diff
@@ -107,7 +107,9 @@ def create_root_element(self):
             from js import iodide
             return iodide.output.element('div')
         except ImportError:
-            return document.createElement('div')
+            div = document.createElement('div')
+            document.body.appendChild(div)
```

Comment:
I believe the comment above indicates that for using this outside of iodide this method should be overwritten. In particular appending a div and the end of the body is not something one would always want to do I imagine.